### DISCUSSION TITLE
How to reset mysql root password

### DISCUSSION THREAD

**Discussion by ifxzero** [leokhoa/laragon#713] [Q&A]
Hi, i am using laragon 6.0 220916 (mysql 8.0.30), i already set root password but somehow i forgot the pass. 
did you guys know how to reset the pass?
i tried editing my.ini files and put new password there but it doesn't work.
![image](https://github.com/leokhoa/laragon/assets/127677231/1e03818b-46a5-4150-af03-63530102492b)
also tried to change root password from laragon menu -> mysql -> change root password
![image](https://github.com/leokhoa/laragon/assets/127677231/076f92a6-0589-4f73-9435-a3d851f1d35a)

thanks in advance

**Comment by mtariqsajid** [comment]
try to run mysql without password checking 

run with mysql without password validation
mysqld --skip-grant-tables


login into mysql
mysql -u root -p
FLUSH PRIVILEGES;
ALTER USER 'root'@'localhost' IDENTIFIED BY 'new_password';

**Reply by ifxzero** [reply]
just triend your solution, it throws this error msg:
mysqld: Can't open shared library 'C:\laragon\bin\mysql\mysql-8.0.30-winx64\lib\plugin\component_reference_cache.dll' (
errno: 126 The specified module could not be found.)
mysqld: Cannot load component from specified URN: 'file://component_reference_cache'.
![image](https://github.com/leokhoa/laragon/assets/127677231/b70e0356-4c88-4cb0-bb32-61dca4c10e7c)

**Reply by mtariqsajid** [reply]
can you try mysqld.exe ?

**Reply by mtariqsajid** [reply]
try this maybe 

https://www.youtube.com/watch?v=rr_hDJLFvQE

**Reply by Onixo-Network** [reply]
> ALTER USER 'root'@'localhost' IDENTIFIED BY 'new_password';

this worked perfect thnx

**Comment by mtariqsajid** [comment]
were you able to fix this issue?

**Comment by dexter74** [comment]
Bonjour,

Je proposes mon astuce.

Fermer Mysql avant toute action.

```
- https://downloads.mysql.com/archives/installer/
- Télécharger la version Identique de sa Base de donnée (Exemple 8.0.30)
- Copier le dossier "C:\Program Files\MySQL\MySQL Server 8.0\lib"
- Coller dans C:\laragon\bin\mysql\mysql-8.0.30-winx64
```

Editer:
```
C:\laragon\bin\mysql\mysql-8.0.30-winx64\my.ini
```

ajouter:
```
skip-grant-tables
```

Lancer :
```
C:\laragon\bin\mysql\mysql-8.0.30-winx64\bin
mysqld --init-file=C:\Reset.txt
```

```
ALTER USER 'root'@'localhost' IDENTIFIED BY 'admin' account unlock;
```

Se connecter en Localhost:3306

**Comment by Herpex** [comment]
Teardown and rebuild the db
